tests/30rooms/11leaving.pl

```diff
@@ -0,0 +1,299 @@
+multi_test "Setup a room, and have the first user leave (SPEC-216)",
+
+    requires => [qw(
+        make_test_room change_room_powerlevels do_request_json_for user
+        more_users can_create_room
+    )],
+
+    provides => [qw( departed_room_id )],
```

Comment:
As this room ID is only used within this file, it saves leaking it into other files (and increasing the interdependency between them) by not putting it in the env., and just storing it as a local variable within the file that the later `test` blocks can read. For prior art, see the use of

  https://github.com/matrix-org/sytest/blob/master/tests/30rooms/08levels.pl#L3